### DISCUSSION TITLE
console Trace/Log

### DISCUSSION THREAD

**Discussion by xu4wang** [directus/directus#11890] [Archive]
Is there a way to have detailed trace/log on the directus console running API?

In ENV file, I've set 
```
LOG_LEVEL="debug"
```

But there is not too much information. Basically I want to check the detail of each incoming HTTP Request/response, as well as the detail of each outgoing HTTP request/response. 

BR,Austin

**Comment by rijkvanzanten** [comment]
Try setting `LOG_LEVEL` to `debug` or `trace`, and set `LOG_STYLE` to `raw` to get the full raw log in JSON, rather than the prettified log line 👍🏻
